src/App.ts

```diff
@@ -800,7 +800,10 @@ function buildSource<IsEnterpriseInstall extends boolean>(
       )['body'];
 
       // When the app is installed using org-wide deployment, team property will be null
-      if (bodyAsActionOrOptionsOrViewActionOrShortcut.team !== null) {
+      if (
+        typeof bodyAsActionOrOptionsOrViewActionOrShortcut.team !== 'undefined' &&
```

Comment:
@stevengill If you prefer `xxx == null`, it sounds fine to me. If we go with this (`== null`), I would like to make the style consistent in this project.

Reply:
This is fine!